tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -6656,7 +6657,6 @@ node_modules/chrome-devtools-frontend/front_end/host/InspectorFrontendHost.js(27
   Index signature is missing in type '{}'.
 node_modules/chrome-devtools-frontend/front_end/host/InspectorFrontendHost.js(407,19): error TS2694: Namespace 'Adb' has no exported member 'Config'.
 node_modules/chrome-devtools-frontend/front_end/host/InspectorFrontendHost.js(445,48): error TS2694: Namespace 'InspectorFrontendHostAPI' has no exported member 'ContextMenuDescriptor'.
-node_modules/chrome-devtools-frontend/front_end/host/InspectorFrontendHost.js(471,12): error TS2538: Type 'string[]' cannot be used as an index type.
```

Comment:
waiting on resolution of index type/type distribution changes -- I think. @weswigham can you confirm?